tensorboard/pip_package/extract_pip_package.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
+tmpdir="$(mktemp -d)"
+tar xzf "$0.runfiles/org_tensorflow_tensorboard/tensorboard/pip_package/pip_packages.tar.gz" \
+    -C "${tmpdir}"
+find "${tmpdir}" | sort
```

Comment:
nit: `-type f`?  Printing the containing directory doesn't seem that useful.

Reply:
I originally had `-type f`, but Colab has a drag-and-drop file upload
interface, so my preferred workflow is: run `:extract_pip_package`,
triple-click the directory name, run `xdg-open <Paste>`, then drag the
appropriate wheel onto the notebook. (Clicking the “Upload” button and
pasting the path in directly requires using a modal dialog, which I’d
rather not do.)